pandas/io/formats/style.py

```diff
@@ -670,8 +697,9 @@ def to_latex(
         ...    ("Numeric", "Integers"): '\${}',
         ...    ("Numeric", "Floats"): '{:.3f}',
         ...    ("Non-Numeric", "Strings"): str.upper
-        ... })
-        >>> s.to_latex()
+        ... })  # doctest: +ELLIPSIS
+        <pandas.io.formats.style.Styler object at 0x...>
+        >>> s.to_latex()  # doctest: +SKIP
```

Comment:
here `s.to_latex()` is generating a string. This might actually be useful to doctest since we are showing the string output here